### PR TITLE
*: Update documents and sqlgrams with "alter" prefixed statements

### PR DESCRIPTION
**Pull request by zimulala**
### What is changed, added or deleted? (Required)

Update documents and sqlgrams with "alter" prefixed statements.
`ALTER TABLE [table_name] ALTER INDEX [index_name] {VISIBLE | INVISIBLE}` isn't support in v4.0. It supports in master. Related to pingcap/tidb#16914

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by zimulala** [comment]
Do I need to remove `sql-statements/sql-statement-alter-index.md` in another PR? This operation support in master and doesn't support in 4.0

File: sql-statements/sql-statement-alter-index.md [hunk 1]
```diff
@@ -1,191 +0,0 @@
----
```
Comment (ran-huang) [inline sql-statements/sql-statement-alter-index.md:1]: Please also delete this document from TOC.

**Comment by ran-huang** [comment]
> Do I need to remove `sql-statements/sql-statement-alter-index.md` in another PR? This operation support in master and doesn't support in 4.0

@zimulala I suggest you adding `sql-statements/sql-statement-alter-index.md`, because all PRs of the special week will be merged into **master**. 
You can create a PR on **release-4.0** branch to remove `sql-statements/sql-statement-alter-index.md` next week. What do you think?

**Comment by zimulala** [comment]
@ran-huang 
I think it's OK. I added this file.

**Pull request merged**